aptos-indexer-processor-example/src/common_steps/latest_processed_version_tracker.rs

```diff
@@ -16,34 +13,28 @@ use aptos_indexer_processor_sdk::{
 use async_trait::async_trait;
 use diesel::{upsert::excluded, ExpressionMethods};
 
-const UPDATE_PROCESSOR_STATUS_SECS: u64 = 1;
+pub const UPDATE_PROCESSOR_STATUS_SECS: u64 = 1;
 
+/// This step tracks the latest version processed by the processor and stores the version in DB.
+/// It assumes that the step inputs are ordered by starting_version.
+/// To guarantee the order by starting_version, you should connect `OrderByStartingVersionStep` before this step.
 pub struct LatestVersionProcessedTracker<T>
 where
     Self: Sized + Send + 'static,
     T: Send + 'static,
 {
     conn_pool: ArcDbPool,
     tracker_name: String,
-    // Next version to process that we expect.
-    next_version: u64,
     // Last successful batch of sequentially processed transactions. Includes metadata to write to storage.
-    last_success_batch: Option<TransactionContext<()>>,
-    // Tracks all the versions that have been processed out of order.
-    seen_versions: AHashMap<u64, TransactionContext<()>>,
-    _marker: PhantomData<T>,
```

Comment:
why does this PR remove PhantomData? Shouldn't this version tracker be the same as the one in `processors` following the addition of the order step?